l10n_mx_toponyms/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Mexican Toponyms
+================
+
+This module extends the functionality of Odoo to add Mexican toponyms according with the catalogues publised by SAT on the CFDI 3.3 specification
+
+Currently this catalogues are supported:
+
+* Paises
+* Estados
+* Colonias
+* Localidades (Ciudades)
+* Municipios 
+* CP
+
+
+Installation
+============
+
+Install this module as any other module on your Odoo instance.
+
+
+Configuration
+=============
+
+No special configuration required.
+
+
+Known issues / Roadmap
+======================
+
+None
+
+Bug Tracker
+===========
+
+Bugs are tracked on `Gitlab Issues
+<http://gitlab.openpyme.mx/l10n_mx/l10n_mx_toponyms/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+* OpenPyme México: `<https://www.openpyme.mx/es>`_.
+
+Contributors
+------------
+
+* Federico Cruz <federico.cruz@openpyme.mx>
+* Agustín Cruz <agustin.cruz@openpyme.mx>
+
+Maintainer
+----------
+
+This module is maintained by OpenPyme.
+
+To contribute to this module, please visit https://gitlab.openpyme.mx.
```

Comment:
@agb80 Please use the template available at 

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

to have:

```
Credits
=======

Images
------

* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.

Contributors
------------

* Federico Cruz <federico.cruz@openpyme.mx>
* Agustín Cruz <agustin.cruz@openpyme.mx>

Maintainer
----------

.. image:: https://odoo-community.org/logo.png
   :alt: Odoo Community Association
   :target: https://odoo-community.org

This module is maintained by the OCA.

OCA, or the Odoo Community Association, is a nonprofit organization whose
mission is to support the collaborative development of Odoo features and
promote its widespread use.

To contribute to this module, please visit https://odoo-community.org.
```